components/Steps/Summary.jsx

```diff
@@ -27,7 +27,14 @@ const Result = ({ formData, clearFormData }) => {
       setError(e.message);
     }
   };
-  if (submitting) return null;
+  if (submitting) {
+    return (
+      <>
+        <h1>Submitting...</h1>
+        <h2>Do not navigate away from this page</h2>
```

Comment:
How about something like the following?

```suggestion
        <h2>Please wait while your application is being submitted</h2>
```

Reply:
https://www.gov.uk/service-manual/design/writing-for-user-interfaces#tone

Reply:
I did not know about this, thank you.